language/move-lang/src/cfgir/liveness/mod.rs

```diff
@@ -294,12 +294,14 @@ mod last_usage {
                     match display_var(v.value()) {
                         DisplayVar::Tmp => (),
                         DisplayVar::Orig(v_str) => {
-                            let msg = format!(
-                                "Unused assignment or binding for local '{}'. Consider removing \
-                                 or replacing it with '_'",
-                                v_str
-                            );
-                            context.error(vec![(l.loc, msg)]);
+                            if !v_str.starts_with('_') {
+                                let msg = format!(
+                                    "Unused assignment or binding for local '{}'. \
+                                    Consider removing or replacing it with '_'",
```

Comment:
We should update this error message to suggest prepending it with `_`

Reply:
Sure, I'll update it. This will lead to several updates to the `exp` files too.